nixos/tests/containers-custom-pkgs.nix

```diff
@@ -0,0 +1,42 @@
+# Test for NixOS' container support.
+
+import ./make-test-python.nix ({ pkgs, lib, ...} : let
+
+  customPkgs = pkgs // {
+    hello = pkgs.hello.overrideAttrs(old: {
```

Comment:
(nitpick - can be ignored optionally) Any reason you're not using an overlay?

Reply:
I didn't want to have to import packages one more time.